sdk/python/kfp/_client.py

```diff
@@ -338,8 +340,69 @@ def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, para
         pipeline_id=pipeline_id,
         workflow_manifest=pipeline_json_string,
         parameters=api_params)
+    return JobConfig(spec=spec, resource_references=resource_references)      
+
+  def run_pipeline_recurring(self, experiment_id, job_name, start_time=None, end_time=None, interval_second=600 , max_concurrency=1, params={}, pipeline_package_path=None, pipeline_id=None, namespace=None, enabled=True):
+     """Create a recurring run.
+    Args:
+      experiment_id: The string id of an experiment.
+      job_name: name of the job.
+      start_time: The time string of the time when to start the job.
```

Comment:
maybe these 2 should be `datetime` and `interval_second` should be a `timedelta`? i don't know whether that's their style but i think it's better.

Reply:
then it doesn't work I think

Reply:
needs to be [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6) since it's a [`date-time`](https://swagger.io/docs/specification/data-models/data-types/#format) in
[kubeflow/pipelines/backend/api/swagger/job.swagger.json](https://github.com/kubeflow/pipelines/blob/570141d7c90f0d999834dfc88ac5110fa780257d/backend/api/swagger/job.swagger.json#L367)